.travis.yml

```diff
@@ -1,22 +1,25 @@
+language: python
 sudo: false
 cache: pip
 
+python:
+- "3.5"
+
 addons:
+  postgresql: "9.5"
   apt:
     packages:
       - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
-
-language: python
-
-python:
-  - "3.5"
 
 env:
-  - VERSION="11.0" LINT_CHECK="1"
-  - VERSION="11.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
-  - VERSION="11.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  global:
+  - VERSION="11.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
+  - PHANTOMJS_VERSION="latest"
```

Comment:
This line is not needed for not adding extra build time.

Reply:
ok